src/librustc_codegen_ssa/traits/statics.rs

```diff
@@ -8,16 +8,15 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use super::Backend;
+use super::BackendTypes;
 use rustc::hir::def_id::DefId;
 use rustc::ty::layout::Align;
 
-pub trait StaticMethods<'tcx>: Backend<'tcx> {
+pub trait StaticMethods: BackendTypes {
```

Comment:
If you want to do this, it should be done to all the traits that right now inherit from `Backend<'tcx>`.